packages/ui/package.json

```diff
@@ -51,6 +51,10 @@
     "normalize.css": "^8.0.1"
   },
   "devDependencies": {
+    "vite": "^2.9.8",
```

Comment:
Let's have deps sorted, how package manager does.

In VS Code select deps, run `Ctrl + Shift + P` and `Sort`